service-admin/src/App/src/Logging/LoggerTrait.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
```

Comment:
Wondering why we need a whole new file here?  Is it copied from elsewhere? I thought we had this in a seperate package that got pulled in by multiple projects?

Reply:
We do, but that LoggerTrait is not usable in service-admin. This is because our LoggerTrait includes code specific to laminas-mvc, which we aren't using in service-admin. Using that LoggerTrait would therefore require adding a dependency on laminas-mvc to service-admin.

The shared LoggerTrait also expects certain headers to be on requests, and looks for events which aren't triggered in service-admin.

I did consider reworking the shared LoggerTrait so it could deal with both the laminas-mvc apps and service-admin. However, as the code is trivial, I decided to implement a simplified LoggerTrait in service-admin, rather than refactor.

Reply:
👍